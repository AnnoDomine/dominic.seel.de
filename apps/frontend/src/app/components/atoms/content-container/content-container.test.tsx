import { render, screen } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import ContentContainer from "./content-container";

describe("ContentContainer", () => {
    it("renders without crashing", () => {
        render(<ContentContainer data-testid="content-container">Content</ContentContainer>);
        const contentContainer = screen.getByTestId("content-container");
        expect(contentContainer).toMatchSnapshot();
    });
});
