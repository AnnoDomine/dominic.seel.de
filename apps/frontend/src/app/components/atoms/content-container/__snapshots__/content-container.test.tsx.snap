// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ContentContainer > renders without crashing 1`] = `
<div
  class="css-13mi9bo-content-container-root"
  data-testid="content-container"
>
  Content
</div>
`;
